examples/advanced/edge/setup_system.md

```diff
@@ -10,28 +10,6 @@ Use tree_prov.py to generate a hierarchical NVFlare system with 2 levels and 2 c
 
 This will create a deployment with 4 leaf nodes, 2 aggregators, 2 relays, and 1 server.
```

Comment:
Please revise:
- Use EdgeJob API to generate job config.
- Use tree_prov to provision the project.
- Use the generated "demo" package to start routing proxy and simulation.

Reply:
done with TP simulation

@YuanTingHsieh will leave routing proxy part to you